src/main/java/ca/yorku/cse2311/tab2pdf/ui/listener/SampleInput_2Listener.java

```diff
@@ -0,0 +1,68 @@
+package ca.yorku.cse2311.tab2pdf.ui.listener;
+
+import ca.yorku.cse2311.tab2pdf.ui.MainJFrame;
+
+import java.awt.Desktop;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+import java.io.IOException;
+import java.net.URI;
+import java.net.URISyntaxException;
+
+/**
+ * HelpListener
+ *
+ * Opens the User Manual for the user to peruse
+ *
+ * @author Anton Sitkovets
+ * @since 2015-03-19
+ */
+ public class SampleInput_2Listener implements ActionListener {
+	 private final String INPUT_FILE_2 = "https://wiki.eecs.yorku.ca/course_archive/2014-15/W/2311/_media/rememberingrain.txt";
+	 private final MainJFrame window;
+    
+
+    /**
+     * @param window    the window we are working with
+     */
+    public SampleInput_2Listener(MainJFrame window) {
+
+        super();
+
+        this.window = window;
+    }
+
+    @Override
+    public void actionPerformed(ActionEvent e) {
+    	open(createURI(INPUT_FILE_2));
+    	
+    }
+    /**
+	 * Opens the URI in the URI object using the desktop's default browser
+	 * @param URI uri
+	 */
+	public static void open(URI uri){
```

Comment:
[DRY](http://en.wikipedia.org/wiki/Don%27t_repeat_yourself) is violated here, perhaps open/createURI should be in some utility class?